docs/reference/unit.md

```diff
@@ -603,9 +619,85 @@ A specialization will only exist if the inputs are all units, and will exist but
 associative, and symmetric under interchange of any inputs.
 
+??? note "A note on inputs vs. outputs for the `common_point_unit(us...)` form"
+    The return value of the instance version is a _unit_, while the input parameters are
+    [unit _slots_](../discussion/idioms/unit-slots.md).  This means that the return value will often
+    be a different _category of thing_ (i.e., always consistently a unit) than the inputs (which may
+    be quantity point makers, unit symbols, or so on).
+
+    For example, consider `common_unit(meters, feet)`.  Recall that the type of `meters` is
+    `QuantityMaker<Meters>`, and that of `feet` is `QuantityMaker<Feet>`.  In this case, the return
+    value is an instance of `CommonUnitT<Meters, Feet>`, _not_
+    `QuantityMaker<CommonUnitT<Meters, Feet>>`.
+
+    If you want something that still computes the common unit, but preserves the _category_ of the
+    inputs, see [`make_common_point(us...)`](#make-common-point).
+
 **Syntax:**
 
 - For _types_ `Us...`:
     - `CommonPointUnitT<Us...>`
+- For _instances_ `us...`:
+    - `common_point_unit(us...)`
+
+## Category-preserving unit slot operations
+
+A [unit slot](../discussion/idioms/unit-slots.md) API can take a variety of "categories" of input.
+Prominent examples include:
+
+- Simple unit types (`Meters{}`, ...)
+- Quantity makers (`meters`, ...)
+- Unit symbols (`symbols::m`, ...)
+- Constants (`SPEED_OF_LIGHT`, ...)
+
+The [previous section](#traits) demonstrated various traits that can be applied to units.  Some of
+these traits (such as `common_unit(...)`) produce a new unit as their output type.  This will always
+be a _simple_ unit, even though the inputs are unit _slots_: that is, these traits change the
+_category_ of the output.
+
+This section describes a few special operations that _preserve_ that category.  So for example,
+suppose we had an operation `op` of this type.  Let's call the result of `op(Meters{}, Seconds{})`
+as `U{}`.  Then we have:
+
+- `op(meters, seconds)` produces `QuantityMaker<U>`, because its inputs are `QuantityMaker<Meters>`
+  and `QuantityMaker<Seconds>`.
+- `op(m, s)` produces `UnitSymbol<U>`, because its inputs are `UnitSymbol<Meters>` and
+  `UnitSymbol<Seconds>`.
+- ... and so on.
+
+Here are the category-preserving operations we provide.
+
+### Making common units {#make-common}
+
+**Result:** A new unit: the largest unit that evenly divides its input units.  (Read more about the
+concept of [common units](../discussion/concepts/common_unit.md).)
+
+**Syntax:**
+
+- `make_common(us...)`
+
+??? example "Examples"
+    ```cpp
+    constexpr auto x = make_common(meters, feet)(18);
+
+    using symbols::m;
+    using symbols::ft;
+    constexpr auto y = 9.5f * make_common(m, ft);
+    ```
+
+### Making common point units {#make-common-point}
+
+**Result:** A new unit: the largest-magnitude, highest-origin unit which is "common" to the input
```

Comment:
In [the test](https://github.com/aurora-opensource/au/pull/362/files#diff-a4df4f6d28089aabc817f1a4fb8860b4ae7efdeebe90424039467d7478685400R576) it says lowest origin. Should this say lowest?

Reply:
No.  :sweat_smile:  But it's really confusingly worded.

To be "common" (for points) to all input units, a unit's origin must _not be higher_ than any of them.  The unit with the _highest_ such origin will have its origin _equal_ to the _lowest origin_ of all input units.

The fix here was to simply abandon this phrasing and rephrase it more carefully.  :slightly_smiling_face: